### ISSUE TITLE
Java 17 is corrupt

### ISSUE THREAD

**Issue by frainie** [YunoHost-Apps/minecraft_ynh#42]
```
2022-06-01 15:40:44,828: DEBUG - + ynh_setup_source --dest_dir=/tmp/tmp.6ev8LLAF6D --source_id=java
2022-06-01 15:40:44,878: DEBUG - + source_id=java
2022-06-01 15:40:44,878: DEBUG - + keep=
2022-06-01 15:40:44,878: DEBUG - + local src_file_path=/var/cache/yunohost/app_tmp_work_dirs/app_r7aq6rxm/conf/java.src
2022-06-01 15:40:44,879: DEBUG - ++ grep SOURCE_URL= /var/cache/yunohost/app_tmp_work_dirs/app_r7aq6rxm/conf/java.src
2022-06-01 15:40:44,879: DEBUG - ++ cut --delimiter== --fields=2-
2022-06-01 15:40:44,882: DEBUG - + local src_url=https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.deb
2022-06-01 15:40:44,883: DEBUG - ++ grep SOURCE_SUM= /var/cache/yunohost/app_tmp_work_dirs/app_r7aq6rxm/conf/java.src
2022-06-01 15:40:44,884: DEBUG - ++ cut --delimiter== --fields=2-
2022-06-01 15:40:44,887: DEBUG - + local src_sum=8f7b6d6d7ba4324f19d94c0a69eef0090280387d09622357c6ec7335656a41ce
2022-06-01 15:40:44,888: DEBUG - ++ grep SOURCE_SUM_PRG= /var/cache/yunohost/app_tmp_work_dirs/app_r7aq6rxm/conf/java.src
2022-06-01 15:40:44,888: DEBUG - ++ cut --delimiter== --fields=2-
2022-06-01 15:40:44,890: DEBUG - + local src_sumprg=sha256sum
2022-06-01 15:40:44,892: DEBUG - ++ grep SOURCE_FORMAT= /var/cache/yunohost/app_tmp_work_dirs/app_r7aq6rxm/conf/java.src
2022-06-01 15:40:44,892: DEBUG - ++ cut --delimiter== --fields=2-
2022-06-01 15:40:44,895: DEBUG - + local src_format=deb
2022-06-01 15:40:44,895: DEBUG - ++ grep SOURCE_EXTRACT= /var/cache/yunohost/app_tmp_work_dirs/app_r7aq6rxm/conf/java.src
2022-06-01 15:40:44,896: DEBUG - ++ cut --delimiter== --fields=2-
2022-06-01 15:40:44,898: DEBUG - + local src_extract=false
2022-06-01 15:40:44,899: DEBUG - ++ grep SOURCE_IN_SUBDIR= /var/cache/yunohost/app_tmp_work_dirs/app_r7aq6rxm/conf/java.src
2022-06-01 15:40:44,900: DEBUG - ++ cut --delimiter== --fields=2-
2022-06-01 15:40:44,902: DEBUG - + local src_in_subdir=
2022-06-01 15:40:44,903: DEBUG - ++ grep SOURCE_FILENAME= /var/cache/yunohost/app_tmp_work_dirs/app_r7aq6rxm/conf/java.src
2022-06-01 15:40:44,903: DEBUG - ++ cut --delimiter== --fields=2-
2022-06-01 15:40:44,907: DEBUG - + local src_filename=jdk-17_linux-x64_bin.deb
2022-06-01 15:40:44,907: DEBUG - + src_sumprg=sha256sum
2022-06-01 15:40:44,907: DEBUG - + src_in_subdir=true
2022-06-01 15:40:44,907: DEBUG - + src_format=deb
2022-06-01 15:40:44,908: DEBUG - ++ echo deb
2022-06-01 15:40:44,908: DEBUG - ++ tr '[:upper:]' '[:lower:]'
2022-06-01 15:40:44,910: DEBUG - + src_format=deb
2022-06-01 15:40:44,911: DEBUG - + src_extract=false
2022-06-01 15:40:44,911: DEBUG - + '[' jdk-17_linux-x64_bin.deb = '' ']'
2022-06-01 15:40:44,911: DEBUG - + local local_src=/opt/yunohost-apps-src/minecraft/jdk-17_linux-x64_bin.deb
2022-06-01 15:40:44,911: DEBUG - + mkdir -p /var/cache/yunohost/download/minecraft/
2022-06-01 15:40:44,912: DEBUG - + src_filename=/var/cache/yunohost/download/minecraft/jdk-17_linux-x64_bin.deb
2022-06-01 15:40:44,912: DEBUG - + test -e /opt/yunohost-apps-src/minecraft/jdk-17_linux-x64_bin.deb
2022-06-01 15:40:44,913: DEBUG - + '[' -n https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.deb ']'
2022-06-01 15:40:44,913: DEBUG - ++ wget --tries 3 --no-dns-cache --timeout 900 --no-verbose --output-document=/var/cache/yunohost/download/minecraft/jdk-17_linux-x64_bin.deb https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.deb
2022-06-01 15:40:50,615: DEBUG - + out='2022-06-01 15:40:50 URL:https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.deb [155551050/155551050] -> "/var/cache/yunohost/download/minecraft/jdk-17_linux-x64_bin.deb" [1]'
2022-06-01 15:40:50,616: DEBUG - + sha256sum --check --status
2022-06-01 15:40:51,697: DEBUG - + ynh_die '--message=Corrupt source'
2022-06-01 15:40:51,720: DEBUG - + ynh_exit_properly
2022-06-01 15:40:51,721: WARNING - Corrupt source
2022-06-01 15:40:53,263: ERROR - Unable to install minecraft: An error occurred inside the app installation script
```
Using the latest version of JunoHost. Appears Java17 has a corrupt source.

**Comment by ericgaspar** [comment]
I don't understand why these sums keep changing 🤔. #44 should fix it

**Comment by frainie** [comment]
Hi, Everything works fine now. Sorry for being a pain but is there a way to access the server console? in order to OP myself and whatnot.

**Comment by Tagadda** [comment]
Closed by #48

**Comment by Tagadda** [comment]
@frainie You can connect to the console via rcon, using a telnet client like Putty.
